packages/server/src/constants/index.ts

```diff
@@ -20,6 +20,7 @@ export enum FilterTypes {
   NOT_EMPTY = "notEmpty",
```

Comment:
I think there may be two of these `FilterTypes` enums - I know this expands *way* beyond what this PR is about, but I'm wondering if we should get rid of the `FilterType` and `FilterTypes` enums and add a single up to date one to types?

Reply:
Yeah, we absolutely should. I'll do that in a follow-up PR if that's alright?